packages/spor-theme-react/src/components/tabs.ts

```diff
@@ -83,11 +83,33 @@ const getTabColorSchemeSelectedProps = (props: StyleFunctionProps) => {
         _focus: {
           boxShadow: `inset 0 0 0 2px ${props.theme.colors.alias.greenHaze}`,
         },
+        "&:focus:not(:focus-visible)": {
+          boxShadow: "none",
+        },
+        _focusVisible: {
+          boxShadow: `inset 0 0 0 2px ${props.theme.colors.alias.greenHaze}`,
+        },
```

Comment:
La til denne linja for å ikke legge til mer focus-styling enn nødvendig i browsers som støtter `:focus-visible`